tests/integration/004_template/main.tf

```diff
@@ -31,6 +31,10 @@ resource "env0_template" "github_template" {
   terraform_version                       = "0.15.1"
 }
 
+data "env0_source_code_variables" "variables" {
+  template_id = data.env0_template.github_template.id
```

Comment:
how about creating an output from the data source ? that's how you can validate integration

Reply:
yes... it depends... see my comment above.

Reply:
added